tomoscan/tomoscan_pso.py

```diff
@@ -291,6 +291,7 @@ def compute_positions_PSO(self):
         # Compute the time for each frame
         time_per_angle = self.compute_frame_time()
         self.motor_speed = np.abs(self.rotation_step) / time_per_angle
+        print(f'{self.rotation_step=}, {time_per_angle=}, {self.motor_speed=}')
```

Comment:
@decarlof remove this

Reply:
addressed [here](https://github.com/tomography/tomoscan/pull/144/commits/76af9722ed7c6a9be60904c7fc3d392720b38e2f)